bin/addNotificationsSettings.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+const admin = require('firebase-admin');
+
+/**
+ * Load service account from SERVICE_ACCOUNT environment variable
+ * or local serviceAccount.json.
+ */
+function loadServiceAccount() {
```

Comment:
Can we move this to a module in `bin` because it will be needed by pretty much every one-off script?

Reply:
Yup, I was actually considering the same

Reply:
Done